sippy-ng/src/component_readiness/TriagedRegressions.js

```diff
@@ -17,7 +17,7 @@ export default function TriagedRegressions(props) {
     })
 
     if (
-      selectedTriagedEntry.regressions !== 'undefined' &&
```

Comment:
this was actually throwing an error in the console that I hadn't noticed, this fixes it